### PR TITLE
Buffing some awful black market items to be less awful.

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
With this PR, I'm making some awful or bad blackmarket items less bad (some are still bad imo). I don't aim to tweak every item that I find mediocre at best, because it ultimately boils down to opinion and maybe what I don't like, others do, but also because the feature was designed to have some "scammy" items in there (the broken chameleon hat is a prime example).

Moving on, some of the more noticeable changes:
- the old spacesuit in a box should no longer cost thousands of credits
- the thermite bottle now contains enough thermite to melt one r-wall
- replaced the shotgun dart item with a more expensive box of XL shotgun darts (25 units of reagent capacity vs 15)
- replaced the science googles with a more expensive medical-security combo HUD (the sprites exist already)
- the donk pocket box item can now spawn subtypes of the donk pocket box, this includes the gondola box, though it's pretty rare.
- the suspicious pill bottle item can now spawn a pill bottle of maintenance pills

The rest should be price changes.

## Why It's Good For The Game
Many of the blackmarkets choices are downright a bummer, and I'm not talking about things like the broken chameleon hat, but stuff such as shotgun darts, science googles, thermite; I mean, SOME of stuff that's OBVIOUSLY easy and more convenient to get from a (proto/auto) lathe or the chemist and just make the blackmarket look kinda bad.

## Changelog

:cl:
balance: reduced the prices of some blackmarket items across the board.
balance: the thermite bottle (from the contraband spawner and the blackmarket), now spawns with 50u of thermite vs 30, enough to melt one reinforced wall.
add: Replaced the science googles from the blackmarket with a security + health scanner HUD.
add: Replaced the single shotgun dart from the blackmarket with a box of XL shotgun darts.
add: The donk pocket box from the blackmarket now comes in different flavors.
/:cl:

### CODE REVIEW

File: code/modules/projectiles/ammunition/ballistic/shotgun.dm [hunk 1]
```diff
@@ -159,7 +159,13 @@
 /obj/item/ammo_casing/shotgun/dart/attackby()
 	return
 
+/obj/item/ammo_casing/shotgun/dart/large
+	name = "XL shotgun dart"
+	desc = "A dart for use in shotguns. Can be injected with up to 23 units of any chemical."
```
Comment (Loafin34) [inline code/modules/projectiles/ammunition/ballistic/shotgun.dm:164]: shouldn't this be 25 units of any chemical?

**Comment by carlarctg** [comment]
this is certainly not a bad thing, but the black market will never be a valuable use of time for the median player if it's just an assorted collection of expensive mediocre loot with a convoluted and annoying recipe.

**Comment by Eronymun** [comment]
> this is certainly not a bad thing, but the black market will never be a valuable use of time for the median player if it's just an assorted collection of expensive mediocre loot with a convoluted and annoying recipe.

Jarvis, add buckshot to the black market.

**Review by ArcaneMusic — APPROVED** [review]
As far as I can see, LGTM.
Prices are fine and science goggles being removed due to having no real purpose for existing is likely of no real consequence.

**Pull request merged**